### DISCUSSION TITLE
How to restrict multiple options of items in the toolbar

### DISCUSSION THREAD

**Discussion by paul-frost** [craftcms/ckeditor#251] [Q&A] [answered]
I'd like to remove the options to choose which type of list or ordered list, as I would prefer to control the style of these things in my CSS.
I would also prefer the Link button to only open the choice menu of linking to an entry, asset or url and not just default to a url. I'm concerned that editors will just click the button and not realise there are other options.

I've searched through the CKEditor docs and can't find any mention of how to do this, but I know from my [query about image placement](https://github.com/craftcms/ckeditor/discussions/243) that this sort of thing is possible. Also, an [example on the docs page about lists doesn't have the option arrow](https://ckeditor.com/docs/ckeditor5/latest/features/lists/lists.html#demo-3).

Where can I find the config options to modify these default button options?

**Comment by joshuabaker** [comment, marked as answer]
The CKEditor v5 docs aren’t quite opaque. I figured this out after trial and error. You want to set `list.properties.styles` to `false`.

Any block/inline styles are applied to nested paragraphs anyway, which is a different gripe. 😒

**Comment by kbergha** [comment]
I just needed the same thing, and figured it out myself before finding the same answer here.

Adding/modifying the config as suggested by @joshuabaker will remove the sub-types of lists.

![2024-09-16 15 05 16 cms cic web oslo kommune localhost b605afb1992b](https://github.com/user-attachments/assets/cac5f356-29f1-4b27-89eb-424ad00048a0)

Note the missing "dropdown arrows" in the editor view:
![2024-09-16 15 07 32 cms cic web oslo kommune localhost 9ca7b8e4ee8e](https://github.com/user-attachments/assets/892efe96-18dc-46a3-9fbe-82779ed84efc)

**Reply by paul-frost** [reply]
Thanks for clarifying how the JSON looks
